dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.jaxws.common-2.3.feature

```diff
@@ -1,46 +1,64 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
-symbolicName=com.ibm.websphere.appserver.internal.jaxws-2.3
+symbolicName=com.ibm.websphere.appserver.jaxws.common-2.3
+Subsystem-Name: Java Web Services Common 2.3
 visibility=private
 singleton=true
+IBM-Process-Types: client, \
+ server
 IBM-App-ForceRestart: uninstall, \
-install
-Subsystem-Name: Internal Java Web Services 2.3
+ install
+IBM-API-Package:\
+ javax.jws; type="spec"; require-java:="9", \
+ javax.jws.soap; type="spec"; require-java:="9", \
+ javax.wsdl; type="spec", \
+ javax.wsdl.extensions; type="spec", \
+ javax.wsdl.extensions.http; type="spec", \
+ javax.wsdl.extensions.mime; type="spec", \
+ javax.wsdl.extensions.schema; type="spec", \
+ javax.wsdl.extensions.soap; type="spec", \
+ javax.wsdl.extensions.soap12; type="spec", \
+ javax.wsdl.factory; type="spec", \
+ javax.wsdl.xml; type="spec", \
+ javax.xml.soap; type="spec"; require-java:="9", \
+ javax.xml.ws; type="spec", \
+ javax.xml.ws.handler; type="spec", \
+ javax.xml.ws.handler.soap; type="spec", \
+ javax.xml.ws.http; type="spec", \
+ javax.xml.ws.soap; type="spec", \
+ javax.xml.ws.spi; type="spec", \
+ javax.xml.ws.spi.http; type="spec", \
+ javax.xml.ws.wsaddressing; type="spec"
 -features=\
- com.ibm.websphere.appserver.injection-1.0, \
- com.ibm.websphere.appserver.servlet-4.0, \
  com.ibm.websphere.appserver.classloading-1.0, \
- com.ibm.websphere.appserver.javax.mail-1.6, \
- com.ibm.websphere.appserver.globalhandler-1.0, \
- com.ibm.websphere.appserver.httpcommons-1.0, \
- com.ibm.websphere.appserver.internal.cxf.common-3.2, \
  com.ibm.websphere.appserver.containerServices-1.0, \
- com.ibm.websphere.appserver.internal.slf4j-1.7.7
+ com.ibm.websphere.appserver.injection-1.0, \
+ com.ibm.websphere.appserver.internal.cxf.common-3.2, \
+ com.ibm.websphere.appserver.httpcommons-1.0, \
+ com.ibm.websphere.appserver.internal.slf4j-1.7.7, \
+ com.ibm.websphere.appserver.javax.mail-1.6, \
+ com.ibm.websphere.appserver.jaxb-2.3
 -bundles=\
+ com.ibm.websphere.javaee.jws.1.0; require-java:="9"; location:="dev/api/spec/,lib/"; mavenCoordinates="javax.jws:jsr181-api:1.0-MR1", \
+ com.ibm.websphere.javaee.jaxws.2.3; location:="dev/api/spec/"; mavenCoordinates="javax.xml.ws:jaxws-api:2.3.0", \
+ com.ibm.websphere.javaee.wsdl4j.1.2; location:="dev/api/spec/,lib/"; mavenCoordinates="wsdl4j:wsdl4j:1.6.3", \
+ com.ibm.websphere.prereq.wsdl4j.api; location:="dev/api/spec/,lib/", \
  com.ibm.ws.com.sun.xml.messaging.saaj; require-java:="9", \
- com.ibm.ws.cxf.client, \
+ com.ibm.ws.javaee.ddmodel.ws, \
```

Comment:
as we talked about with xmlWS-3.0 ddmodel.ws is only in server and not common with the client.